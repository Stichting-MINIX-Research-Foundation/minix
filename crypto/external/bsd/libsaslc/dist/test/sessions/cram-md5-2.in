CRAM-MD5
2
AUTHID none
PASSWD blablablablablablabla
<1896.697170952@postoffice.reston.mci.net>
