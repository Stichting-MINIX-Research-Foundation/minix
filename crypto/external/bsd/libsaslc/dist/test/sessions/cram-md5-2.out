none 4aa7fd2cc3ea8a69033de42fede35476
